go/pkg/filemetadata/cache.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var globalCache cache.SingleFlight
 
-// Reset clears the cache globally.
+// ResetGlobalCache clears the cache globally.
```

Comment:
You can enable this linter check to prevent this kind of mismatch.
https://golangci-lint.run/usage/linters/#stylecheck